pkgs/by-name/fo/forgejo-cli/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+  pkg-config,
+  libgit2,
+  oniguruma,
+  openssl,
+  zlib,
+  stdenv,
+  darwin,
+}:
+let
+  version = "0.1.1";
+in
+rustPlatform.buildRustPackage {
+  pname = "forgejo-cli";
+  inherit version;
```

Comment:
```suggestion

rustPlatform.buildRustPackage rec {
  pname = "forgejo-cli";
  version = "0.1.1";
```

Reply:
Won't eval; is missing this suggestion^

Reply:
Oh wait, nevermind, it does. Guess you prefer `let in` over `rec`.
Any particular reason?

Reply:
> Any particular reason?

Personal preference really. I mean I created this issue out of curiosity what others think https://github.com/NixOS/nixpkgs/issues/315337